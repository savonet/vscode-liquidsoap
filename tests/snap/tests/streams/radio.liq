# Basic radio test

mic = sine()
bed = mksafe(playlist("playlist"))

radio = add([mic, bed])

output.dummy(fallible=true, radio)

def on_done () =
  test.pass()
  shutdown()
end

thread.run(delay=10., on_done)
