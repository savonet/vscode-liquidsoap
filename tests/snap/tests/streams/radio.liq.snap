># Basic radio test
#^^^^^^^^^^^^^^^^^^^ source.liquidsoap comment.line.number-sign.liquidsoap
>
>mic = sine()
#^^^^^^^^^^^^^ source.liquidsoap
>bed = mksafe(playlist("playlist"))
#^^^^^^^^^^^^^^^^^^^^^^ source.liquidsoap
#                      ^ source.liquidsoap string.quoted.double.liquidsoap
#                       ^^^^^^^^ source.liquidsoap string.quoted.double.liquidsoap
#                               ^ source.liquidsoap string.quoted.double.liquidsoap
#                                ^^^ source.liquidsoap
>
>radio = add([mic, bed])
#^^^^^^^^^^^^^^^^^^^^^^^^ source.liquidsoap
>
>output.dummy(fallible=true, radio)
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.liquidsoap
>
>def on_done () =
#^^^ source.liquidsoap keyword.control.liquidsoap
#   ^^^^^^^^^^^^^^ source.liquidsoap
>  test.pass()
#^^^^^^^^^^^^^^ source.liquidsoap
>  shutdown()
#^^^^^^^^^^^^^ source.liquidsoap
>end
#^^^ source.liquidsoap keyword.control.liquidsoap
>
>thread.run(delay=10., on_done)
#^^^^^^^^^^^^^^^^^ source.liquidsoap
#                 ^^ source.liquidsoap constant.numeric.integer
#                   ^^^^^^^^^^^^ source.liquidsoap
>